credsweeper/filters/value_grafana_check.py

```diff
@@ -0,0 +1,38 @@
+import base64
+import contextlib
+import json
+
+from credsweeper.credentials import LineData
+from credsweeper.filters import Filter
+
+
+class ValueGrafanaCheck(Filter):
+    """Grafana Provisioned API Key and Access Policy Token"""
+
+    def run(self, line_data: LineData) -> bool:
+        """Run filter checks on received token which might be structured.
+
+        Args:
+            line_data: credential candidate data
+
+        Return:
+            True, when need to filter candidate and False if left
+
+        """
+        if not line_data.value:
+            return True
+        with contextlib.suppress(Exception):
+            if line_data.value.startswith("glc_"):
+                # Grafana Access Policy Token
+                decoded = base64.b64decode(line_data.value[4:])
+                keys = ["o", "n", "k", "m"]
+            else:
+                # Grafana Provisioned API Key
+                decoded = base64.b64decode(line_data.value)
+                keys = ["n", "k", "id"]
```

Comment:
Could you give me a example of this case?
I can't find out from sample for this case.

Reply:
The code is for next PR, just use here to reduce merge  conflicts

Reply:
https://github.com/Samsung/CredSweeper/pull/312